core/Updates/4.0.0-b1.php

```diff
@@ -153,6 +155,10 @@ public function getMigrations(Updater $updater)
             $migrations[]     = $this->migration->db->sql("UPDATE $visitActionTable SET search_count = custom_var_v5 WHERE custom_var_k5 = '_pk_scount'");
         }
 
+        $logVisit = Common::prefixTable('log_visit');
+        // until 3.14.0 config_cookie was always 0 if cookies were disabled so for most visits this will be fairly accurate.
+        $migrations[] = $this->migration->db->sql("UPDATE $logVisit SET profilable = config_cookie WHERE profilable is null");
```

Comment:
It be a problem if someone was using Matomo 3.14.0 for a long time, then upgraded to Matomo 4. Then this may not be as accurate. Could also remove this update script.

Reply:
it might be fine to leave it, but it would be important to document it in the developer changelor or in the changelog somewhere. btw would it be maybe possible to update only up to the time that they had upgraded to 3.14.0?

Reply:
AFAIK we don't know when they upgraded